### DISCUSSION TITLE
Syncing always get interrupted in 2min ~ 30min

### DISCUSSION THREAD

**Discussion by Photonico** [abraunegg/onedrive#1659] [Q&A] [answered]
Hi,

Onedrive version is v2.4.13, and OS is openSUSE.
Everytime I sync with `onedrive --synchronize`, it will be interrupted in 2min ~ 30min.
How to fix it?

Thanks

**Comment by jayctheriot** [comment]
If you are doing this through wifi or USB check the power-settings on the
device.  Windows, Linux and Apple vary on how to do this, and the words
they use.  Basically, you want performance, prevent sleep, etc. settings
like that.

I know occasionally on long transfers, my local connection will go to
sleep.  This isn't a onedrive issue, per se.  But settings on the host OS
not realizing there is data transfer going on.

I've witnessed the behavior on multiple platforms and operating systems.
Disabling power management, generally helps.

Jay C. "Jazzy J" Theriot

**Comment by abraunegg** [comment, marked as answer]
@Photonico 
> Everytime I sync with onedrive --synchronize, it will be interrupted in 2min ~ 30min.
> How to fix it?

The sync will get interrupted if your network connection to OneDrive gets interrupted.

Investigate your network connectivity.
